og_migrate/migrations/d7_og_user_membership.yml

```diff
@@ -0,0 +1,55 @@
+id: d7_og_user_membership
+label: Group user memberships
+migration_tags:
+  - Drupal 7
+source:
+  plugin: d7_og_membership
+  entity_type: user
+process:
+  id: id
+  type:
+    -
+      plugin: migration_lookup
+      migration: d7_og_membership_type
+      source: type
+    -
+      plugin: skip_on_empty
+      method: row
+      message: 'Membership type is missing'
+  uid:
+    -
+      plugin: migration_lookup
+      migration: d7_user
+      source: etid
+    -
+      plugin: skip_on_empty
+      method: row
+      message: 'User uid is missing'
+  entity_id: gid
+  entity_type:
+    -
+      plugin: og_entity_type_exists
+      source: group_type
+      bundle_property: entity_bundle
+    -
+      plugin: skip_on_empty
+      method: row
+      message: 'Entity type is missing'
+  roles:
```

Comment:
The entity_bundle field seems to be missing here?

Reply:
This seems to be set by the og_entity_type_exists plugin used for entity_type.
The plugin sets the value for the destination property 'entity_bundle'.